ui/perfherder/compare/CompareTable.jsx

```diff
@@ -28,14 +30,44 @@ export default class CompareTable extends React.PureComponent {
       displayNumber(percentage),
     )}% ${improvement ? 'better' : 'worse'})`;
 
+  // humane readable signature name
+  getSignatureName = (testName, platformName) =>
+    [testName, platformName].filter(item => item !== null).join(' ');
+
+  getHashBasedId = (testName, platformName = null) => {
+    const { hashFunction } = this.props;
+
+    const tableSection = platformName === null ? 'header' : 'row';
+    const hashValue = hashFunction(
+      this.getSignatureName(testName, platformName),
+    );
+
+    return `comptable-${tableSection}-${hashValue}`;
```

Comment:
You could also just use `table` since compare-view is in the url name :)

Reply:
Indeed.